Cargo.toml

```diff
@@ -53,6 +53,14 @@ arrow-array = { version = "42.0.0", default-features = false, features = ["chron
 parquet = { version = "42.0.0", features = ["arrow", "async", "object_store"] }
 sqlparser = { version = "0.35", features = ["visitor"] }
 
+[patch.crates-io]
```

Comment:
I think we will have to wait for this code to be released to crates.io (in about 10 days) before merging this PR